ui/app/AppLayouts/Wallet/views/walletconnect/WalletConnectSDK.qml

```diff
@@ -1,326 +1,338 @@
-            root.runJavaScript(jsCode,
-                function(result) {
-                    timer.restart()
-                }
-            )
+        function onSessionExpire(details)
+        {
+            console.debug(`@dd TODO WalletConnectSDK.onSessionExpire; details: ${JSON.stringify(details, null, 2)}`)
         }
 
-        function startListeningForEvents() {
-            const jsCode = "
-                try {
-                    function processWCEvents() {
-                        window.wcEventResult = [];
-                        window.wcEventError = null
-                        window.wc.registerForSessionRequest((event) => {
-                            window.wcEventResult.push({name: 'session_request', payload: event});
-                        });
-                    }
-                    processWCEvents();
-                } catch (e) {
-                    window.wcEventError = e
-                }
-                window.wcEventError"
+        function onSessionRequest(details)
+        {
+            console.debug(`@dd WalletConnectSDK.onSessionRequest; details: ${JSON.stringify(details, null, 2)}`)
+            root.sessionRequestEvent(details)
+        }
 
-            root.runJavaScript(jsCode,
-                function(result) {
-                    if (result) {
-                        console.error("startListeningForEvents: processWCEvents error", result)
-                        return
-                    }
-                }
-            )
-            eventsTimer.start()
+        function onSessionRequestSent(details)
+        {
+            console.debug(`@dd TODO WalletConnectSDK.onSessionRequestSent; details: ${JSON.stringify(details, null, 2)}`)
         }
 
-        function init(projectId) {
-            d.requestSdkAsync(generateSdkCall("init", `"${projectId}"`, RequestCodes.SdkInitSuccess, RequestCodes.SdkInitError))
+        function onSessionEvent(details)
+        {
+            console.debug(`@dd TODO WalletConnectSDK.onSessionEvent; details: ${JSON.stringify(details, null, 2)}`)
         }
 
-        function getPairings(projectId) {
-            d.requestSdk("getPairings", `null`, RequestCodes.GetPairings, RequestCodes.GetPairingsError)
+        function onProposalExpire(details)
+        {
+            console.debug(`@dd WalletConnectSDK.onProposalExpire; details: ${JSON.stringify(details, null, 2)}`)
+            root.pairSessionProposalExpired()
         }
     }
 
     ListModel {
         id: pairings
     }
-}
\ No newline at end of file
+
+    WebChannel {
+        id: statusChannel
+        registeredObjects: [statusObject]
+    }
+
+    WebEngineView {
+        id: webEngineView
+
+        anchors.fill: parent
+
+        url: "qrc:/app/AppLayouts/Wallet/views/walletconnect/sdk/src/index.html"
```

Comment:
Why did you remove the possibility of loading the SDK from the disk for quick prototyping with the storybook? I suggest keeping it until we finish the implementation. I found it helpful for rapid prototyping in this hard-to-debug environment.

Reply:
As discussed, we will keep it for now and handle that when we get to the storybook.